docs/concepts/pools/deploy-a-pool.md

```diff
@@ -0,0 +1,39 @@
+# Pool Creation
+
+## Overview
+
+**Note: Before creating a new pool, make sure there isn't already a similar pool. There is no advantage to fragmenting liquidity.**
+
+Anyone can create a pool on Balancer. WeightedPools can be deployed using the Pool creator Interface that can be accessed on [Mainnet](https://app.balancer.fi/#/ethereum), [Polygon](https://app.balancer.fi/#/polygon), [Arbitrum](https://app.balancer.fi/#/arbitrum), and [Gnosis Chain](https://app.balancer.fi/#/gnosis-chain). It's highly recommended that you deploy a pool on a testnet before deploying on a production network.
+
+## What Type of Pool Should I Deploy?
+
+The type of pool you deploy depends on your use case. After reading througt the descriptions of different Balancer pool types, decide which type fits best your use case.
+
+## Deployment Process Summary
+
+### Step1: Deploy the Pool from Factory
+
+Each _poolType_ has a factory from which users can deploy new pools. To deploy a pool, you must call that factory's _create()_ function with the arguments that correspond to that specific pool. Please check the "Common Arguments" section below for frequently used arguments.
+
+### Step 2: Add liquidity with the INIT join
+
+The INIT join can be done only once when the pool have a BPT totalSupply of 0. Almost all pools require you to use a JoinKind of type INIT before you can use the pool.
```

Comment:
Again for a UI user, this is all non-sense.  For a techincal user, there's a lot of information missing.

- Approvals have to be in place first
- The init join sets the initial price and should be weighted as such0
- At best some screenshots or examples of an init join operation on common pool types (ComposableStable, Weighted)

This is a concept doc, we should help technically minded  people understand what they are doing and why it is necessary.